### PR TITLE
Always use LTR layout

### PR DESCRIPTION
**Pull request by mhmdanas**
ViMusic does not support any languages other than English (at least for the near future, see https://github.com/vfsfitvnm/ViMusic/pull/171#issuecomment-1206528846), and if the system language uses an RTL script (e.g. Arabic or Hebrew), the UI is broken in many places. For instance:

RTL bug demonstration

Tested and confirmed to be working.

### CODE REVIEW

File: app/src/main/kotlin/it/vfsfitvnm/vimusic/ui/components/themed/DropdownMenu.kt [hunk 1]
```diff
@@ -133,23 +133,15 @@ private data class DropdownMenuPositionProvider(
...
-        } else {
-            sequenceOf(
-                toLeft,
-                toRight,
-                // If the anchor gets outside of the window on the right, we want to position
-                // toDisplayRight for proximity to the anchor. Otherwise, toDisplayLeft.
-                if (anchorBounds.right <= windowSize.width) toDisplayLeft else toDisplayRight
-            )
-        }.firstOrNull {
+        .firstOrNull {
```
Comment (mhmdanas) [inline app/src/main/kotlin/it/vfsfitvnm/vimusic/ui/components/themed/DropdownMenu.kt:144]: I'm fine with reverting this change too, if you wish.

**Comment by vfsfitvnm** [comment]
Good spot.
I think disabling the RTL support in the manifest is the right and only thing to do. A RTL user will contribute and make the required adjustments, eventually.

**Comment by vfsfitvnm** [comment]
I apologize for not merging your PR - I didn't want to have an extra commit to revert the changes made to DropdownMenu.kt

**Comment by mhmdanas** [comment]
> A RTL user will contribute and make the required adjustments, eventually.

Indeed, but only if/when support for other languages is added. There's no point in using RTL if it's with English; it would only makes the app look strange.

**Comment by vfsfitvnm** [comment]
> You should've been able to remove that change and then force-push to my branch, but that's fine with me too, no worries.

Uh, thanks, I didn't know it was possible! I should've looked online before assuming it wasn't possible :face_exhaling:

**Pull request closed without merging**